docs/Concepts/Node-Types.md

```diff
@@ -96,6 +97,32 @@ You can't switch from fast sync to snap sync.
 If your node is blocked in the middle of a fast sync, you can start over using snap sync instead by stopping the node,
 deleting the data directory, and starting over using `--sync-mode=X_SNAP`.
 
+### Checkpoint synchronization
+
+!!! important
+
+    Checkpoint sync is an experimental feature.
+
+Enable checkpoint sync using [`--sync-mode=X_CHECKPOINT`](../Reference/CLI/CLI-Syntax.md#sync-mode).
+You need Besu version 22.4.3 or later to use checkpoint sync.
+
+Checkpoint sync behaves like [snap sync](#snap-synchronization), but instead of syncing from the genesis block, it syncs
+from a specific checkpoint configured in the [Besu genesis file](../HowTo/Configure/Genesis-File.md).
+Checkpoints are currently defined in the network configurations for Ethereum Mainnet and the Ropsten and Goerli testnets.
+
+!!! example "Ethereum Mainnet checkpoint configuration"
```

Comment:
This configuration has already been defined in Besu, so users will automatically get it when using for example, `--network ropsten --sync-mode=X_CHECKPOINT`. The example makes it seem like you need to manually add it.

We may need to make this more clear (if my assumptions are correct)